lib/middlewares/alexa.ts

```diff
@@ -1,17 +1,59 @@
-import AlexaVerifier from 'alexa-verifier-middleware';
+import verifier from 'alexa-verifier';
+import { NextFunction, Request, Response } from 'express';
 
+import log from '@/logger';
 import { Config } from '@/types';
 
 import { FullServiceMap } from '../services';
 import { AbstractMiddleware } from './utils';
 
+export const AlexaVerifierMiddleware = (req: Request & { _body: any; rawBody: any }, res: Response, next: NextFunction) => {
+  if (req._body) {
+    const er = 'The raw request body has already been parsed.';
+    res.status(400).json({ status: 'failure', reason: er });
+    return;
+  }
+
+  // TODO: if _rawBody is set and a string, don't obliterate it here!
+
+  // mark the request body as already having been parsed so it's ignored by
+  // other body parser middlewares
+  req._body = true;
+  req.rawBody = '';
+  req.on('data', (data: string) => {
+    // eslint-disable-next-line no-return-assign
+    return (req.rawBody += data);
+  });
+
+  req.on('end', () => {
+    try {
+      req.body = JSON.parse(req.rawBody);
+    } catch (parseError) {
+      log.error('body parse rejection: %s, %s', parseError, req.rawBody);
```

Comment:
the voiceflow/logger package hasn't been updated here so this still compiles.
the logger accepts a single string argument, ex.
```suggestion
      log.error(`body parse rejection:${parseError}, ${req.rawBody}`);
```

Reply:
I see, we should update it everywhere else